src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setColor(new ColorSupplier().getRandomColor());
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
The same applies to the `Random` instance. You should avoid creating a new `Random` instance every time the method is called. Instead, create a single `Random` instance as a class field and reuse it.